test/e2e/network/service.go

```diff
@@ -1217,6 +1217,37 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(err)
 	})
 
+	/*
+			Create a ClusterIP service with an Exteranl IP that is not assigned to an interface.
```

Comment:
typo `Exteranl` and tab different that the line below :)

Reply:
Thanks!

Reply:
squash commits please, and we should be ready to go

Reply:
All set